### PR TITLE
Add NetBSD ki_sigset_t, kinfo_proc2, and kinfo_lwp structs

### PR DESCRIPTION
**Pull request by Freaky**

### CODE REVIEW

File: libc-test/semver/netbsd.txt [hunk 1]
```diff
@@ -1275,8 +1275,10 @@ jrand48
 kevent
 key_t
 killpg
-kinfo_vmentry
```
Comment (Amanieu) [inline libc-test/semver/netbsd.txt:1278]: Why was this removed?

Comment (Freaky) [reply libc-test/semver/netbsd.txt:1278]: It wasn't removed, I just sorted this hunk - it's under kinfo_proc2.
